.changeset/README.md

```diff
@@ -0,0 +1,25 @@
+# Changeset sjabloon
+
+Kopieer en plak het onderstaande sjabloon. Je kunt hiervoor de kopieer knop linksboven in het template gebruiken.
+
+```markdown
+---
+'@<scope>/<package-name>': major
+---
+
+Beschrijving
+```
+
+Vul tussen de twee sets `---` in voor welke packages je allemaal veranderingen hebt doorgevoerd. Zet elk package tussen
+dubbele aanhalingstekens op een nieuwe regel.
+
+Gebruik:
+
+- `"@<scope>/<package-name>": major` voor breaking changes
+- `"@<scope>/<package-name>": minor` voor nieuwe features
+- `"@<scope>/<package-name>": patch` voor bug fixes
```

Comment:
misschien kunnen we hier beter `@example/<package-name>` doen, omdat we de instructie hebben om find-replace te doen op `@example/`